test-requirements.txt

```diff
@@ -1,5 +1,7 @@
-pytest>=2.8.7
-pytest-cov>=2.2.1
+coverage~=4.3.4
+codecov~=2.0.5
+pytest~=3.0
+pytest-cov~=2.2
 pytest-env>=0.6.0
 pytest-timeout>=1.0.0
-pytest-xdist>=1.14
\ No newline at end of file
+pytest-xdist>=1.14
```

Comment:
I'd prefer to keep them the same as coala core repo, and definitely that is a subject for a different PR.